MoneySplitter.Win10/Common/SearchEngine.cs

```diff
@@ -0,0 +1,90 @@
+﻿using MoneySplitter.Models;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Windows.UI.Xaml;
+using Caliburn.Micro;
+using MoneySplitter.Infrastructure;
+using System.Linq;
+
+namespace MoneySplitter.Win10.Common
+{
+    public class SearchEngine : PropertyChangedBase
+    {
+        private readonly DispatcherTimer _timer;
+
+        private readonly ISearchApiService _searchApiService;
+        private bool _isSearchInProgress;
+        private string _previousQuery = string.Empty;
+        private string _query;
+        private const int INTERVAL_TIME = 1;
+
+        private TimeSpan TIMER_INTERVAL = TimeSpan.FromSeconds(INTERVAL_TIME);
```

Comment:
Don't create consts for another consts. In this case just use TimeSpan.FromSeconds(1);